scripts/lint.ts

```
@@ -0,0 +1,559 @@
+
+	cache.reconcile();
+}
+
+export function runOxlint(
+	filePath: string,
+	extraFlags: Array<string> = [],
+	runner = DEFAULT_SETTINGS.runner,
+): string | undefined {
+	const flags = extraFlags.length > 0 ? `${extraFlags.join(" ")} ` : "";
+	try {
+		execSync(`${runner} oxlint ${flags}"${filePath}"`, {
```

ESLint

[error] Unsafe call of a type that could not be resolved.

(ts/no-unsafe-call)